DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Centre/Administrator/AdministratorControllerTests.cs

```diff
@@ -188,5 +188,36 @@ public void UnlockAccount_unlocks_account_and_returns_to_page()
             A.CallTo(() => userDataService.UpdateAdminUserFailedLoginCount(1, 0)).MustHaveHappened();
             result.Should().BeRedirectToActionResult().WithActionName("Index");
         }
+
+        [Test]
+        public void DeactivateAdminUser_does_not_deactivate_admin_user_without_confirmation()
+        {
+            // Given
+            var adminUser = new AdminUser() { Id = 1 };

```

Comment:
Minor, but it's probably best to be consistent with other tests and use `UserTestHelper.GetDefaultAdminUser()` for the adminUser definition in these new tests.

Reply:
Used GetDefaultAdminUser method as suggested